slp2mp4/slp-to-mp4.py

```diff
@@ -129,12 +142,15 @@ def record_folder_slp(slp_folder, conf):
     if len(out_files) == 0:
         RuntimeError("No slp files in folder!")
     last_dir = out_files[0][0]
+    args = []
     for index, in_file in enumerate(in_files):
 
         # Combine last subdirectory on discovery of a new subdirectory
         if out_files[index][0] != last_dir:
             if conf.combine:
-                combine(conf)
+                #combine(conf)
+                # TODO: Find a way to fix combining folders while still multiprocessing
```

Comment:
Easiest way to do this is probably multiprocess one directory at a time

Reply:
Can be saved for another PR though

Reply:
That's the easiest way to do it, but it impacts throughput because you
- Can only record up to the number of games in the directory instead of what can be supported by the system. E.g. if my computer can run 4 games at a time but we have two folders of 2 .slp files, what we have now can record them ~twice as fast.
- You have to wait for the last game to end before starting to record the next directory. The last game in the recording process always slows throughput because only one game is recording at that time.

For fastest throughput, we would have something that detects when all the valid games are in the output folder and then combine them, but it has to happen while the multiprocessing is running.

I agree that it should wait until another PR.